README.md

```diff
@@ -25,14 +25,14 @@ documentation].
 |:-------:|:---------------:|:------------:|:-------------:|
 |   v1.4  |     [v1.4.3]    | Aug 31, 2021 | March 2022    |
 |   v1.5  |     [v1.5.1]    | Oct 26, 2021 | May 2022      |
-|   v1.6  |     [v1.6.0]    | Jan 4, 2022  | July 2022     |
+|   v1.6  |     [v1.6.1]    | Jan 4, 2022  | July 2022     |
 |   v1.7  |     Upcoming    | Mar 22, 2022 | Sept 2022     |
 
 You can subscribe to [release calendar] to track all release dates.
 
 [v1.4.3]: https://github.com/crossplane/crossplane/releases/tag/v1.4.3
 [v1.5.1]: https://github.com/crossplane/crossplane/releases/tag/v1.5.1
-[v1.6.0]: https://github.com/crossplane/crossplane/releases/tag/v1.6.0
+[v1.6.1]: https://github.com/crossplane/crossplane/releases/tag/v1.6.1
```

Comment:
How valuable to folks feel linking from this table to the latest patches is? I like the table capturing the set of supported minor versions, but it's a bit of a pain to have to open a PR here for patch releases given that (IMO) the most recent patch release for a particular minor version is _fairly_ easy to find using the releases page.